### ISSUE TITLE
flickrannex re-uploading already uploaded images

### ISSUE THREAD

**Issue by magthe** [TobiasTheViking/flickrannex#8]
I've noticed that sometimes `flickrannex` re-uploads an image (I'm only using it for images so far) that has already been uploaded to flickr.  I've been able to reproduce it, but I haven't got a clue as to why.  It only happens when the number of images becomes fairly large, it didn't occur at ~350 images, but does occur at ~530.

To reproduce it I've preformed the following actions:
1. Create an annex with more than 500 images (534 in my case)
2. Copy the images to an empty flickr account by running `git annex copy . -t flickr`.
3. Check the number of images in the `gitannex` set, note that it matches the number of files in the annex minus any copy failures.
4. Re-run `git annex copy . -t flickr`.
5. Check the number of images in the 'gitannex' set.

I expected that only the images that failed to copy the first time would be copied the second time, but that wasn't the case.  In my case the first copy operations failed on 1 image, and the second copy operation still copied 35 images!

**Comment by TobiasTheViking** [comment]
I have way too little information to help you.

Could you please redownload the latest version from github (Verify it is version 0.1.5) and replicate?

To get debug information from flickrannex change your flickr-hook to

```
flickr-hook = /usr/bin/python2 /my/path/flickrannex/flickrannex.py --dbglevel 1 --dbg --stderr 
```

**Comment by TobiasTheViking** [comment]
This issue should be fixed as of version 0.1.6
